blocks/data-cards/data-cards.css

```diff
@@ -0,0 +1,50 @@
+.data-cards>ul {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 10px; /* Added gap here for spacing between cards */
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+}
+
+.data-cards>ul>li {
+
+    border-radius: 10px 0 0 0;
+    border: 4px solid transparent;
+    width: 402px;
+    height: 388px;
```

Comment:
Please don't add fixed height. This will limit the text that we can author in the card.